app/src/main/java/pl/futurecollars/invoicing/db/file/FileBasedDatabase.java

```diff
@@ -43,8 +44,8 @@ public int save(Invoice invoice) {
         idService.setId();
```

Comment:
nazwa setId() może być trochę myląca - nie wiadomo na co ustawiamy wartość. Zmieniłbym nazwę na coś co mówi o zachowaniu metody (szczególnie o tym, że robimy tam inkrementację, bo to nie jest oczywiste jak się patrzy na ten serwis)